### PR TITLE
fix SearchFilter.vue

### PR DESCRIPTION
**Pull request by ToBinio**
fixes the SearchFilter so it can start

not sure what happened but whenever I open the [page](https://omorphia.modrinth.com/components/search-filter.html) just "crashes"

maybe I am doing something wrong?

### CODE REVIEW

File: lib/components/search/SearchFilter.vue [hunk 1]
```diff
@@ -40,11 +40,11 @@ const props = defineProps({
   },
 })
 
-const isActive = ref(props.activeFilters.value.includes(props.facetName))
+const isActive = ref(props.activeFilters.includes(props.facetName))
 const emit = defineEmits(['toggle'])
 
 watchEffect(() => {
-  isActive.value = props.activeFilters.value.includes(props.facetName)
+  isActive.value = props.activeFilters.includes(props.facetName)
```
Comment (brawaru) [inline lib/components/search/SearchFilter.vue:47]: I don't see `isActive` be changed anywhere else. I think `isActive` should be a computed ref instead.

```js
const isActive = computed(() => props.activeFilters.includes(props.facetName))
```

**Review by brawaru — CHANGES_REQUESTED** [review]
Left a comment on relevant code, LMK if I'm wrong.

File: lib/components/search/SearchFilter.vue (hunk 1 above)
Comment (ToBinio) [reply lib/components/search/SearchFilter.vue]: tbh, I didn't even know what `compute` and had to read the docs... 
but makes sense

File: lib/components/search/SearchFilter.vue [hunk 2]
```diff
@@ -40,11 +40,11 @@ const props = defineProps({
...
 })
 
-const isActive = ref(props.activeFilters.value.includes(props.facetName))
+const isActive = computed(() => props.activeFilters.includes(props.facetName))
 const emit = defineEmits(['toggle'])
 
 watchEffect(() => {
-  isActive.value = props.activeFilters.value.includes(props.facetName)
+  isActive.value = props.activeFilters.includes(props.facetName)
 })
```
Comment (brawaru) [inline lib/components/search/SearchFilter.vue:48]: This block can be removed, because it does essentially what `computed` does.

**Review by brawaru — CHANGES_REQUESTED** [review]

File: lib/components/search/SearchFilter.vue (hunk 2 above)
Comment (ToBinio) [reply lib/components/search/SearchFilter.vue]: makes even more sense...

**Review by brawaru — APPROVED** [review]
Looks good!

**Review by Geometrically — APPROVED** [review]

**Pull request merged**